### ISSUE TITLE
My save changes dont show up in game

### ISSUE THREAD

**Issue by Wenno1208** [marcrobledo/savegame-editors#121]
when i edit a savefile and swap out my save file in botw when i load the save file non of my changes show and i spawn in the shrine of resurection

**Comment by marcrobledo** [comment]
Make sure you are replacing the files correctly and are not creating copies of the old ones. The savegame folder and file structure must remain the same.

**Comment by Wenno1208** [comment]
yeah i just tried to do that but its still not working by the way i am using cemu

**Comment by marcrobledo** [comment]
Start a new game.
Open the savegame in the editor.
Do not do make change and save.
If it restarts you in the shrine, then it's not due to the editor.
If it doesn't, try to open the savegame in the editor, make a single change and run the game again. Repeat until you find out which edit is breaking your savegame.

**Comment by atlasbotch** [comment]
hello, im having a similar problem, (btw im on switch) i edited my save and replaced to it to all of my saves but when i load to my game it doesnt change, can you please help me
